neptune/internal/backends/alpha_integration_backend.py

```diff
@@ -122,25 +184,101 @@ def create_experiment(self,
+            ch_values = [
+                alpha_operation.LogStrings.ValueType(
+                    value=value.y.get('text_value'),
+                    step=None,
+                    ts=int(value.ts),
+                )
+                for value in channel_with_values.channel_values
+            ]
+            send_operations.append(alpha_operation.LogStrings(
+                path=alpha_path_utils.parse_path(channel_with_values.channel_id),
+                values=ch_values,
+            ))
+
+        self._execute_alpha_operation(experiment, send_operations)
+
+    @with_api_exceptions_handler
     def get_system_channels(self, experiment):
-        return list()
+        params = {
+            'experimentId': experiment.internal_id,
+        }
+        try:
+            experiment = self.leaderboard_swagger_client.api.getExperimentAttributes(**params).response().result
+        except HTTPNotFound:
+            # pylint: disable=protected-access
+            raise ExperimentNotFound(
+                experiment_short_id=experiment.id, project_qualified_name=experiment._project.full_id)
+        return [
+            AlphaChannelWithLastValue(
+                ch_id=attr.stringSeriesProperties.attributeName,
+                ch_name=alpha_path_utils.parse_path(attr.stringSeriesProperties.attributeName)[-1],
+                ch_type=attr.stringSeriesProperties.attributeType,
+            )
+            for attr in experiment.attributes
+            if (attr.type == AlphaAttributeType.STRING_SERIES.value
+                and attr.name.startswith(MONITORING_ATTRIBUTE_SPACE))
+        ]
 
+    @with_api_exceptions_handler
     def create_system_channel(self, experiment, name, channel_type):
-        return NonCallableMagicMock()
+        channel_id = f'{MONITORING_ATTRIBUTE_SPACE}{name}'
+        dummy_log_string = alpha_operation.LogStrings(
+            path=alpha_path_utils.parse_path(channel_id),
+            values=[],
+        )  # this operation is used to create empty attribute
+        self._execute_alpha_operation(
+            experiment=experiment,
+            operations=[dummy_log_string],
+        )
+        system_channels = self.get_system_channels(experiment)
+        for channel in system_channels:
+            if channel.name == name:
+                return channel
+        raise ChannelNotFound(channel_id=channel_id)
 
     def upload_experiment_source(self, experiment, data, progress_indicator):
+        # TODO: handle `FileChunkStream` or update `neptune.experiments.Experiment._start`
         pass
 
     @with_api_exceptions_handler
     def get_experiment(self, experiment_id):
-        return NonCallableMagicMock()
+        experiment = super().get_experiment(experiment_id)
+        fake_experiment = NonCallableMagicMock()
+        # `timeOfCreation` is required by `TimeOffsetGenerator`
+        fake_experiment.timeOfCreation = dateutil.parser.parse(experiment.creationTime)
```

Comment:
Nothing more is required?

Reply:
Not yet, but in future we'll have to figure out where to get missing information from - new `getExperiment` endpoint doesn't return some of the values, old one returned.
[Usage of get_experiment](https://github.com/neptune-ai/neptune-client/blob/dev/client-integration-with-alpha/neptune/experiments.py#L166).

Reply:
From `getExperimentAtributes` I guess. But it is not to be considered in this merge request, is it?

Reply:
> But it is not to be considered in this merge request

No, it's not.